src/main/java/com/flansmod/common/teams/TeamsManager.java

```diff
@@ -192,6 +192,24 @@ public void tick() {
             FlansMod.getPacketHandler().sendToAll(new PacketTeamInfo());
             showTeamsMenuToAll(true);
         }
+        //check players positions every second.
+        if(time % 20 == 0){
+            if(instance.currentRound!=null && instance.currentRound.map!=null){
+                if(instance.currentRound.teams!=null) {
+                    EntityPlayer player;
+                    for (Team team : instance.currentRound.teams) {
+                        if(team!=null && team.members!=null && !team.members.isEmpty()){
```

Comment:
!team.members.isEmpty() is unneeded, if there's no items in the array then the for loop won't execute anything.